js/components/DeploymentComponent.jsx

```diff
@@ -1,13 +1,15 @@
 /** @jsx React.DOM */
 
+var lazy = require("lazy.js");
```

Comment:
Arh, Lazy is used as a function in stead of a library. Ok, well then I guess it makes sense